clusters/mainnet-staging-na/mainnet-citus/helmrelease.yaml

```
@@ -40,6 +40,14 @@ spec:
   values:
     importer:
       env:
+        #  Speed up historical sync
+        HEDERA_MIRROR_IMPORTER_DOWNLOADER_RECORD_BATCHSIZE: "600"
+        HEDERA_MIRROR_IMPORTER_DOWNLOADER_RECORD_FREQUENCY: "1"
+        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_ENTITY_REDIS_ENABLED: "false"
+        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_FREQUENCY: "10"
+        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_QUEUECAPACITY: "40"
+        ####
+
```

I'm not sure if # or newlines causes any rendering issues. Safer to omit them.

```suggestion
        HEDERA_MIRROR_IMPORTER_DOWNLOADER_RECORD_BATCHSIZE: "600"
        HEDERA_MIRROR_IMPORTER_DOWNLOADER_RECORD_FREQUENCY: "1"
        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_ENTITY_REDIS_ENABLED: "false"
        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_FREQUENCY: "10"
        HEDERA_MIRROR_IMPORTER_PARSER_RECORD_QUEUECAPACITY: "40"
```